utopiaReservation/config.js

```diff
@@ -0,0 +1,12 @@
+require('dotenv').config();//instatiate environment variables
+
+let CONFIG = {} //Make this global to use all over the application
+
+CONFIG.db_dialect   = process.env.DB_DIALECT
+CONFIG.db_host      = process.env.DB_HOST
+CONFIG.db_port      = process.env.DB_PORT
```

Comment:
Good job on implementing environment variables to store sensitive information. LGTM!